src/Games/NexusMods.Games.StardewValley/StardewValley.cs

```diff
@@ -48,6 +48,19 @@ public override GamePath GetPrimaryFile(GameStore store)
         );
     }
 
+    protected override Version GetVersion(GameLocatorResult installation)
+    {
+        try
+        {
+            var fileInfo = new GamePath(LocationId.Game, "Stardew Valley.dll").Combine(installation.Path).FileInfo;
```

Comment:
Seems a bit convoluted to create the absolute path like that.
Shouldn't it just be: `installation.Path.Combine("Stardew Valley.dll").FileInfo:` ?

Reply:
I just realized that this was also wrong, it's `Stardew Valley.dll` for Windows and Linux but macOS uses `Contents/MacOS/Stardew Valley.dll`.